frontend/src/features/properties/filter/PropertyFilter.test.tsx

```diff
@@ -28,15 +28,18 @@ const getStore = (filter: any) =>
   });
```

Comment:
I'll make a story for myself to fix up these unit tests next sprint, I'll need some help and no one is really around right now